doc/10-icinga-template-library.md

```diff
@@ -2861,6 +2861,44 @@ ping_packets                     | **Optional.** The number of packets to send.
 ping_timeout                     | **Optional.** The plugin timeout in seconds. Defaults to 0 (no timeout).
 
 
+### Log Files <a id="plugins-contrib-log-files"></a>
```

Comment:
Please move this section into the `Log Management` section, imho no need to duplicate this.